cmd/soroban-cli/src/config/sign_with.rs

```diff
@@ -101,4 +117,48 @@ impl Args {
             .account(&self.source_account)?
             .public_key(self.hd_path)?)
     }
+
+    pub fn sign_tx_env_with_lab(
+        &self,
+        network: &Network,
+        tx_env: &TransactionEnvelope,
+    ) -> Result<(), Error> {
+        if !self.yes {
```

Comment:
Just wanted to flag this - do we still want to do this for `sign-with-lab`? I think that we would run into a similar issue that @willemneal fixed in https://github.com/stellar/stellar-cli/pull/1590 with the stdin reader. 

More context: https://github.com/AhaLabs/stellar-cli/pull/25#discussion_r1755504162

Reply:
It's removed!